src/runtime/metatype.cs

```diff
@@ -147,37 +148,26 @@ public static IntPtr tp_new(IntPtr tp, IntPtr args, IntPtr kw)
             flags |= TypeFlags.HaveGC;
             Util.WriteCLong(type, TypeOffset.tp_flags, flags);
 
-            TypeManager.CopySlot(base_type, type, TypeOffset.tp_dealloc);
+            //TypeManager.CopySlot(base_type, type, TypeOffset.tp_dealloc);
 
             // Hmm - the standard subtype_traverse, clear look at ob_size to
             // do things, so to allow gc to work correctly we need to move
             // our hidden handle out of ob_size. Then, in theory we can
             // comment this out and still not crash.
-            TypeManager.CopySlot(base_type, type, TypeOffset.tp_traverse);
-            TypeManager.CopySlot(base_type, type, TypeOffset.tp_clear);
-
+            //TypeManager.CopySlot(base_type, type, TypeOffset.tp_traverse);
+            //TypeManager.CopySlot(base_type, type, TypeOffset.tp_clear);
```

Comment:
Please, remove dead code entirely.

Can you also comment why is this no longer necessary?

Reply:
These slots can inherit from the base type automatically. Emmm, I can add some comments here, but it still needs the dead code comment as an explanation.

Reply:
The comment above also seems dangerously wrong.

Reply:
Hmmm, that was an old comment, that time before we unify the shutdown progresses, it can't and won't use tp_traverse and tp_clear. Seems it can be removed at one time.

Reply:
Any dead code should be removed immediately.